package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "jadelet",
-  "version": "0.8.0-pre.7",
+  "name": "jadelet-et",
```

Comment:
Probably don't want to name it "jadelet-et" :P

Reply:
Why not? :P Fixing!